From: Andres Salomon <dilinger@queued.net>
Date: Thu, 17 Nov 2022 01:27:39 -0500
Subject: yt-dlp

Patch to invoke yt-dlp instead of youtube-dl because it's the non-rusty fork.
See https://bugs.debian.org/1024214
===================================================================
---
 src/mainactions.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mainactions.cpp b/src/mainactions.cpp
index 8067888..ffc58d1 100644
--- a/src/mainactions.cpp
+++ b/src/mainactions.cpp
@@ -233,7 +233,7 @@ void ytdl::printResult(int result_num) {
 }
 
 void ytdl::downloadAction() {
-    std::string ytdl_prog = "youtube-dl 2> /tmp/ytdl_stderr --no-warnings --all-subs";
+    std::string ytdl_prog = "yt-dlp 2> /tmp/ytdl_stderr --no-warnings --all-subs";
     std::string url_str = quote + QString_to_str(ui->lineURL->text()) + quote;
     std::string directory_str = quote + QString_to_str(ui->lineBrowse->text()) + "/%(title)s.%(ext)s" + quote;
     std::string parse_output = R"(stdbuf -o0 grep -oP '^\[download\].*?\K([0-9]+)')";
